pkg/kubecli/vmi.go

```diff
@@ -246,7 +247,21 @@ func (v *vmis) VNC(name string) (StreamInterface, error) {
 	return v.asyncSubresourceHelper(name, "vnc")
 }
 func (v *vmis) SerialConsole(name string) (StreamInterface, error) {
-	return v.asyncSubresourceHelper(name, "console")
+	con, err := v.asyncSubresourceHelper(name, "console")
+	for err != nil {
```

Comment:
Isn't it an issue that the function may never exit? The change may hang functional tests indefinitely, among other things.